shift_detector/Detector.py

```diff
@@ -28,14 +28,14 @@ def __init__(self,
         if type(df1) is pd.DataFrame:
```

Comment:
What about this TODO

Reply:
Speeds up debugging but needs to be removed later